config/application.rb

```diff
@@ -21,6 +21,8 @@ class Application < Rails::Application
     config.action_controller.action_on_unpermitted_parameters = :raise
     config.middleware.insert_before ActionDispatch::ParamsParser, "CatchApiJsonParseErrors"
 
+    config.active_record.schema_format = :sql
```

Comment:
Yes. we have to setup triggers to create the tsvector columns. Also I'm not sure why we were avoiding it other than aesthetic reasons? Our code isn't portable to any other databases.